contracts/modules/staking/StakingPool.sol

```diff
@@ -1223,16 +1216,54 @@ contract StakingPool is IStakingPool, ERC721 {
     return ownerOf(0);
   }
 
-  /* management */
+  /* pool management */
+
+  function setProducts(ProductParams[] memory params) external onlyManager {
+    _setProducts(params);
+  }
+
+  function _setProducts(ProductParams[] memory params) internal {
+    uint32 _targetWeight = targetWeight;
+    for (uint i = 0; i < params.length; i++) {
```

Comment:
This loop benefits from more spacing to make it more readable

Check out the other loops in StakingPool.sol for how we've been doing it

https://github.com/NexusMutual/smart-contracts/blob/13f9ab594f5c0e1fe4d0b67cf48004c79763d6b2/contracts/modules/staking/StakingPool.sol#L359